packages/glsp-sprotty/src/types.ts

```diff
@@ -48,8 +48,5 @@ export const TYPES = {
     IDiagramOptions: Symbol('IDiagramOptions'),
     IDiagramStartup: Symbol('IDiagramStartup'),
     IToolManager: Symbol('IToolManager'),
-    IDebugManager: Symbol('IDebugManager'),
-    Grid: Symbol('Grid'),
-    IGridManager: Symbol('IGridManager'),
-    IChangeBoundsManager: Symbol('IChangeBoundsManager')
+    Grid: Symbol('Grid')
```

Comment:
Shouldn't this be IGrid, for consistency?

Reply:
I'd rather not create an interface `IGrid` since it is an alias of `Point` which also does not use the `I`-prefix that is not that common in Typescript anyway. Unless you have very strong feelings about that.